website/docs/error-reference.md

```diff
@@ -925,9 +923,9 @@ end
 
 You can choose the valid layers using the `--packager-layers` command line flag.
 For example, the following specifies that there are three valid layers: `util`,
-`lib` and `app`, ordered lowest to highest.
-
-<!-- TODO(neil): explain what lowest to highest means once we implement these checks -->
+`lib` and `app`, ordered lowest to highest. This means that in `layered` (or
+stricter), packages in the `app` layer can import those in `util` or `lib`, but
+packages in the `util` layer cannot import those in `lib` or `app`.
```

Comment:
Let's move the "This means that..." bit to after the codeblock, so that "This" refers to the code in the codeblock (precise) vs the previous sentence (imprecise).

We probably also want to use the words "the order matters" here somewhere in the explanation of how this `--package-layers` flag works.